game_session.js

```diff
@@ -63,4 +66,48 @@ module.exports = class GameSession {
     getDefaultBeginningCutoffYear() {
         return BEGINNING_SEARCH_YEAR;
     }
+
+    resetGender() {
+        this._gameOptions.gender = [ FEMALE ];
+    }
+
+    setGender(genderArr) {
+        // Return true when gender is successfully updated, false otherwise
+        if (genderArr.length === 0) {
+            this.resetGender();
+        }
+        else {
+            genderArr.map((gender, i) => { genderArr[i] = genderArr[i].toLowerCase(); })
```

Comment:
`map()` doesn't mutate, and you're not doing anything with the return value

Reply:
Are you sure? I'm mutating the array directly. Before this, entering something like `MALE` wouldn't be identified.

Reply:
Didn't see you were modifying the array contents in the callback, looks fine